README.md

```diff
@@ -36,11 +43,39 @@ python manage.py import_languages
 python manage.py import_instruments
 python manage.py download_imgs
 ```
+
 > Note: `download_imgs` might take a while depending on your internet connection. In case the download is interrupted, run the command again inside the container shell.
 
 The Django development server should now be available at `localhost:8000`. Ensure that the database schema is properly set up and the application can display data as expected by navigating to `localhost:8000/instruments/`.
 
-NOTE: that it is recommended to bring the containers down with `docker compose down` after you are done developing, testing, and committing your changes. After executing the previous command, the application will no longer be available `localhost:8000`. To once again view the running application, run `docker compose up -d` and ensure all migrations  made by you or other developers are up-to-date with `docker compose exec -it app bash` and `python manage.py migrate`. For further clarifications on managing migrations refer to this [section](#managing-database-migrations).
+NOTE: that it is recommended to bring the containers down with `docker compose down` after you are done developing, testing, and committing your changes. After executing the previous command, the application will no longer be available `localhost:8000`. To once again view the running application, run `docker compose up -d` and ensure all migrations made by you or other developers are up-to-date with `docker compose exec -it app bash` and `python manage.py migrate`. For further clarifications on managing migrations refer to this [section](#managing-database-migrations).
+
+### UI Set-up Instructions
```

Comment:
I don't love that this is running outside a Docker container for development. In development, some static files are being served through nginx and some aren't. I think we need to choose one or the other, and as far as nginx goes, we need to choose to route through nginx. If we choose not Docker, then let's get rid of Docker.